test/NelsonSiegelSvensson.jl

```diff
@@ -0,0 +1,27 @@
+@testset "NelsonSiegelSvensson" begin
+    
+    @testset "NelsonSiegel" begin
+
+        # EURAAA_20191111
```

Comment:
just to confirm - are these test targets from an external validation source (e.g. a textbook, whitepaper, etc.)? It's not clear to me if there is independence of the validating numbers or if that's just the answer from running the code.

Reply:
They are just results from running the code but looking at the curve generated from the parameters they look very close to the actual data. Will try to add more data to test.

Reply:
Added more tests.